doc/changelog/07-commands-and-options/10494-diffs-in-show-proof.rst

```diff
@@ -0,0 +1,5 @@
+- Highlight the differences between successive proof steps in the
+  "Show Proof" command if the "Set Diffs" option is enabled.  Experimental;
```

Comment:
```suggestion
  :cmd:`Show Proof` command if the :flag:`Diffs` flag is enabled.  Experimental;
```

Reply:
Ah, diffs is an option ("on", "off", "removed"), not a flag.

Reply:
Right, sorry.